sentry/table_sentry_issue_alert.go

```diff
@@ -0,0 +1,195 @@
+			},
+			{
+				Name:        "filter_match",
+				Type:        proto.ColumnType_STRING,
+				Description: "Trigger actions if all, any, or none of the specified filters match.",
+			},
+			{
+				Name:        "frequency",
+				Type:        proto.ColumnType_INT,
+				Description: "Perform actions at most once every X minutes for this issue. Defaults to 30.",
+			},
+			{
+				Name:        "task_uuid",
+				Type:        proto.ColumnType_STRING,
+				Description: "The UUID of the async task that can be spawned to create the issue alert.",
+				Transform:   transform.FromField("TaskUUID"),
+			},
+			{
+				Name:        "actions",
+				Type:        proto.ColumnType_JSON,
+				Description: "List of actions.",
+			},
+			{
+				Name:        "conditions",
+				Type:        proto.ColumnType_JSON,
+				Description: "List of conditions.",
+			},
+			{
+				Name:        "created_by",
+				Type:        proto.ColumnType_JSON,
+				Description: "The rule creator who created the issue alert.",
+			},
+			{
+				Name:        "filters",
+				Type:        proto.ColumnType_JSON,
+				Description: "List of filters.",
+			},
+			{
+				Name:        "projects",
+				Type:        proto.ColumnType_JSON,
+				Description: "The projects for which the issue alert is created.",
+			},
+
+			/// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: "Title of the resource.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+		},
+	}
+}
+
+type IssueAlert struct {
+	sentry.IssueAlert
+	OrganizationSlug string
+	ProjectSlug      string
+}
+
+func listIssueAlerts(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	project := h.Item.(*sentry.Project)
+	projectSlug := d.EqualsQuals["project_slug"].GetStringValue()
+
+	// check if the provided projectSlug is not matching with the parentHydrate
+	if projectSlug != "" && projectSlug != project.Slug {
+		return nil, nil
+	}
+
+	conn, err := getClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("sentry_issue_alert.listIssueAlerts", "connection_error", err)
+		return nil, err
+	}
+
+	params := &sentry.ListCursorParams{}
+	for {
+		issues, resp, err := conn.IssueAlerts.List(ctx, *project.Organization.Slug, project.Slug, params)
+		if err != nil {
+			plugin.Logger(ctx).Error("sentry_issue_alert.listIssueAlerts", "api_error", err)
+			return nil, err
+		}
+		for _, issue := range issues {
+			d.StreamListItem(ctx, IssueAlert{*issue, *project.Organization.Slug, project.Slug})
+
+			// Context can be cancelled due to manual cancellation or the limit has been hit
+			if d.RowsRemaining(ctx) == 0 {
+				return nil, nil
+			}
+		}
+
+		if resp.Cursor != "" {
+			params.Cursor = resp.Cursor
+		} else {
+			break
+		}
+	}
+
+	return nil, nil
+}
+
+func getIssueAlert(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	orgSlug := d.EqualsQuals["organization_slug"].GetStringValue()
```

Comment:
```suggestion
	orgSlug := d.EqualsQualString["organization_slug"]
```
@bigdatasourav I believe we could make this change across all the qual references.

Reply:
Updated with d.EqualsQualString()